doc/userguide/rules/sip-keywords.rst

```diff
@@ -177,3 +183,129 @@ Example
 ::
 
   sip.protocol; content:"SIP/2.0"
+
+sip.from
+--------
+
+This keyword matches on the From field that can be present in SIP headers.
```

Comment:
What happens if there are multiple `From` headers ?

Should it be a multi buffer ?

Reply:
Yes, it should.